src/controllers/statisticsController.js

```diff
@@ -0,0 +1,104 @@
+import models from '../db/models';
+
+const { Statistics } = models;
+
+/**
+ * @description class responsible for statistics of author's article readCount
+ *
+ * @class StatisticsController
+ */
+class StatisticsController {
+  /**
+   * @description - This method is responsible for counting updating article readCount
+   *
+   * @static
+   * @param {object} request - Request sent to the router
+   * @param {object} author - Username of the author of the article
+   *
+   * @returns {object} - object representing response message
+   *
+   * @memberof StatisticsController
+   */
+  static async hasReadArticle(request, author) {
+    const articleSlug = request.params.slug;
+
+    const lastRead = await Statistics.findOne({
+      where: { articleSlug },
+      order: [['id', 'DESC']]
```

Comment:
Hi @marcdomain, I would like to know why you are ordering the response when it is just one element that you are returning here...

Reply:
I meant to use findAll. Thanks for noticing this potential bug.